workloads/kube-burner/README.md

```diff
@@ -2,12 +2,12 @@
 
 The purpose of these scripts is to run a kube-burner workload steered by ripsaw. There are 3 types of workloads at the moment:
 
-- **`cluster-density`**
-- **`node-density`**
-- **`node-density-heavy`**
-- **`max-namespaces`**
-- **`max-services`**
-- **`pod-density`**
+- **`cluster-density`**: Triggered by `run_clusterdensity_test_fromgit.sh`
+- **`node-density`**: Triggered by `run_nodedensity_test_fromgit.sh`
+- **`node-density-heavy`**: Triggered by `run_nodedensity-heavy_test_fromgit.sh`
+- **`max-namespaces`**: Triggered by `run_maxnamespaces_test_fromgit.sh`
+- **`max-services`** Triggered by `run_maxservices_test_fromgit.sh`
+- **`pod-density`**: Triggered by `run_poddensity_test_fromgit.sh`
```

Comment:
If the name of the test is `pod-density` I figure it makes sense to call it `run_pod-density.sh`.

Also if all of them are tests from git, then the suffix `_test_fromgit` doesn't really differentiate them? Did we have a different source (not `fromgit`) or do we plan to?

Reply:
yeah, you're right. I didn't update these odd file names to prevent breaking backwards compatiblity